src/main/java/com/connexta/ddf/catalog/direct/MetacardMap.java

```diff
@@ -14,15 +15,23 @@
 
 public class MetacardMap {
 
-  public static Map<String, Object> convert(Metacard metacard) {
+  private static final String LISTS = "lists";
```

Comment:
could make this public so it can be used here https://github.com/connexta/ddf-jsonrpc/pull/19/files#diff-4b8f31c7553886d543ac3f68641f3f4eR590 without redefining it